wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+
+- Decent online connection.
+- A osu!account logged into Bancho (should be done automatically during in-game osu!account sign-in)
+- Knowing [how to transfer beatmaps without leaving the room.][Beatmaps2 wikilink].
+
+### Ways to enter
+
+From the main menu:-
+
+- Press `Play` then `Multi`; or
+- Press `P` then `M`. (from Keyboard)
+
+If the connection request with Bancho was successful, the player will be directed to the **lobby**, the core of the Multi-play.
+
+### Notes
+
+Keep in mind that downloading times can range from a few seconds to 15 minutes onwards depending on the player's connection speed.
+
+Losing connection to Bancho will direct the player back to main menu until the connection to Bancho has been recovered.
+When playing a match, a "Quit" will be given on the player box.
+
+If the player is in the **cuttingedge** build, the in-game osu!account must have an active [osu!supporter][osu!supporter wikilink] tag to enter.
+
+## Lobby
+
+![Example of Multi's Lobby screen][Multi_lobby image]
+
+_A typical day in the Lobby screen._
+
+The lobby will show all the currently available Match Setup in Bancho.
+Sort out by game mode or fine-tune the search by filtering or searching for a player's name.
+The [Chat Console][Chat Console wikilink] will be available at the bottom of the screen and connected to `#lobby` by default.
+
+---
+
+Each row contains a Match Setup, with the explanation split between left and right side for easier reading:-
+
+- On the left side, there is the game mode symbol, game mode name with (Team Mode) enclosed in brackets, number of players inside over number of players allowed, and rank difference from highest to lowest
+- On the right side, there is a large host profile image and smaller players profile image placement in the Match Setup along with the Match Setup's name and the beatmapset's name with \[beatmap/difficulty\] enclosed in square brackets to be played for this match.
+
+To enter a Match Setup, click on one of the desired row.
+
+**Note:** If the game mode symbol contains a lock icon, clicking on it will prompt an empty text box to enter the Match Setup password.
+The player should enter the correct password to access the locked Match Setup.
+Alternatively, the player can request an invite from the Match Setup's host directly to bypass the locked password requirement.
+
+---
+
+A new Match Setup can be created by pressing the `New Game` button.
+
+The `Quick Join` button will direct the player to an available Match Setup randomly based on current rank range.
+
+Click on the `Back to Menu` button to return to the main menu.
+
+## New game
+
+![New Game setup screen][Multi_new_game image]
+
+_Setting up a new game_
+
+Title | Description
+:---: | :---
+**Game Name**   | **Name of the room.** Default to `{accountname}`'s game. Foreign characters allowed (will be parsed). To make the Match Setup private, enable the `Require Password to join` button.
```

Comment:
`Foreign characters` = unsafe ASCII? (just wondering myself but you probably don't need to note this)